knowledge-base/adr/0028-public-api-extraction.md

```diff
@@ -52,21 +56,35 @@ Hence, we propose the following approach:
      This removes the internal API form the `d.ts` files.
   2. Maintain minimal named exports in the root `index.ts` pointing to the objects of the public API.
      This creates also minimal module exports in the JavaScript usecase.
+  3. We create a `internal` export as we did for the [v4 case](https://github.com/SAP/cloud-sdk-js/tree/v1.28.0/packages/core) which can be imported via `@sap-cloud-sdk/odata-v2/internal`.
+     This internal package contains all exports.
```

Comment:
`internal` is a folder right? The word `package` is a little confusing here.

Reply:
yes